apps/labs/posts/python-wheels-from-tags-to-variants.md

```diff
@@ -0,0 +1,1083 @@
+---
+title: 'Python Wheels: from Tags to Variants'
+published: August 1, 2025
+authors: [michal-gorny]
+description: 'The story of how the Python Wheel Variants design was developed'
+category: [Packaging]
+featuredImage:
+  src: /posts/python-wheels-from-tags-to-variants/featured.jpeg
+  alt: A black steam train pulling a olive-colored carr in reverse. It has cool wheels with red spikes. Photograph by the author.
+hero:
+  imageSrc: /posts/python-wheels-from-tags-to-variants/hero.jpeg
+  imageAlt: A black steam train pulling a olive-colored carr in reverse. It has cool wheels with red spikes. Photograph by the author.
+---
+
+# Python Wheels: from Tags to Variants
+
+Many Python distributions are uniform across different Python versions
+and platforms. For these distributions, it is sufficient to publish
+a single binary package that can be installed everywhere. However, some
+packages are more complex than that — they include compiled Python
+extensions, binaries or even Python code that differs across systems.
+In order to robustly deploy these software on different platforms,
+you need to publish multiple binary packages, with the installers
+being able to select the one best fit the platform used.
+
+For a long time, [Python Wheels](https://packaging.python.org/en/latest/specifications/binary-distribution-format/)
+made do with a relatively simple mechanism
+of providing the needed variance: [Platform Compatibility Tags](https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/).
```

Comment:
"providing the needed variance" is a little awkward / wordy. A minimal change could be "describing the needed variance"?